patternlab/styleguide/source/_patterns/03-organisms/by-template/sticky-toc.twig

```diff
@@ -8,7 +8,7 @@
     <div class="ma__sticky-toc__container">
       <h2 class="ma__sticky-toc__title" id="page-contents">{{ stickyTOC.title }}</h2>
       <nav class="ma__sticky-toc__links" aria-labelledby="page-contents">
-        <button class="ma__sticky-toc__toggle-link">+</button>
+        <button class="ma__sticky-toc__toggle-link"></button>
         <div class="ma__sticky-toc__column">
           <div class="ma__sticky-toc__secondary-label">{{ stickyTOC.title }} <span class="secondary-label-close"></span></div>
```

Comment:
<img width="421" alt="Mass_Gov_1" src="https://user-images.githubusercontent.com/9633303/63112511-6bec7380-bf5e-11e9-8ae6-9ae27c074535.png">

The close button doesn't seem to get focus via keyboard and screen readers since its `<span>`. 
This is an assumption since I cannot test this part.  Here are the expected presentation and behaviors.

1. the close button needs to be focusable and recognized as a button by AT devices.
    - use `<button>` instead of `<span>` if no issues by doing so.

1. it needs its label. So, it needs the similar treatment as **.ma__sticky-toc__toggle-link** above.
   - remove the pseudo content "x" from **.ma__sticky-toc__secondary-label .secondary-label-close::after**. Screen readers read it, but doesn't deliver helpful information to users.
   - put "x" in the button wrapping with `<span aria-hidden="true">`, so screen readers don't announce it.
   - add visually hidden text to the button:
```<span aria-hidden="true">x</span><span class="ma__visually-hidden">Hide the table of contents</span>```



1. As the close button is clicked, the focus moves back on the  `<button class="stickyTOC-open">`.

Reply:
Move this issue to DP-15605.